lib/index.js

```diff
@@ -562,6 +562,33 @@ class Instagram {
       `accounts/access_tool/current_follow_requests?__a=1${cursorParam}`
     )
   }
+  
+  async getUserMediaTagged({id, first, after}){
+    return this.request.get('/graphql/query/', {
+      qs: {
+        query_hash: 'ff260833edf142911047af6024eb634a',
+        variables: JSON.stringify({
+          id: id,
+          first: first,
+          after: after
+        })
+      }
+    })
+    .then(data => data.data.user["edge_user_to_photos_of_you"])
```

Comment:
It is safe to chain data to avoid errors. As we don't have optional channing I'd use
`    .then(response => response.data)
    .then(data=> data.user)
    .then(user=> user.edge_user_to_photos_of_you)`